frameworks/person-escort-record/questions/religious-cultural-needs.yml

```
@@ -0,0 +1,23 @@
+type: radio
+question: Do they have any religious or cultural needs?
+hint: For example, they may be fasting or require a prayer mat.
+description: Religious or cultural needs
+prefill: false
```

I think this should be

```suggestion
prefill: true
```

There might be a bit of confusion between the use of `prefill` here and being "prefilled from NOMIS". Having looked at the ticket I think there were questions whether we could get this information from NOMIS but the NOMIS support is only related to alerts where, if present, they will be displayed alongside the question regardless of the answer.

`prefill: true` will allow this question to be populated *only* from a previous confirmed record which I think we do want in this instance? Unless there is a reason we want them to answer this fresh for each record @ElizabethGoddard?

---

Ah! That makes sense, I had assumed `prefill` referred to the NOMIS functionality rather than the ability for a PER to be populated based on a previous record. I think you're right that this needs to be `true`.

---

Thanks @teneightfive yes I agree I don't see why it can't be populated from a previous PER, it will still have the blue highlight to show it needs to be reviewed right?

---

>it will still have the blue highlight to show it needs to be reviewed right?

Yep, that's correct. Any time there is an answer from a previous assessment it will be displayed in this style :)